roles/ceph-mon/tasks/openstack_config.yml

```diff
@@ -1,11 +1,7 @@
 ---
 - name: create openstack pool
   command: ceph --cluster {{ cluster }} osd pool create {{ item.name }} {{ item.pg_num }}
-  with_items:
-    - "{{ openstack_glance_pool }}"
-    - "{{ openstack_cinder_pool }}"
-    - "{{ openstack_nova_pool }}"
-    - "{{ openstack_cinder_backup_pool }}"
+  with_items: openstack_pools
```

Comment:
It's 2.1.1.0. Perhaps it displayed a deprecation warning and I didn't pay attention to it. I will update the PR. Thanks.